rmgpy/species.py

```diff
@@ -488,6 +488,16 @@ def is_surface_site(self):
         """Return ``True`` if the species is a vacant surface site."""
         return self.molecule[0].is_surface_site()
 
+    def get_binding_sites(self):
```

Comment:
Number of binding sites is in fact a property of the molecule. Perhaps the method should live in that Class.
You're just assuming here that the first molecule is the representative one.
(Once we get surface resonance working, it might make a difference...?)

Reply:
That's a good suggestion. I can move the method. Surface resonance does not change the number of binding sites, so we can use the first molecule to check it.